### PR TITLE
added collision maps to results event tab

### LINKED ISSUE #511: Link to map with collision position
From events list, potentially show the two cars using the relative information on the map too.

### PR DESCRIPTION
**Pull request by Hecrer**
Fixes #511

### CODE REVIEW

File: cmd/server-manager/typescript/src/Results.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    private static processCollisionForm(e: JQuery.SubmitEvent): boolean {
+        e.preventDefault();
+        e.stopPropagation();
+
+        let formArray = $(e.currentTarget).serializeArray();
+
+        let collisions = "";
+
+        for (let i = 0 ; i < formArray.length ; i++) {
```
Comment (cj123) [inline cmd/server-manager/typescript/src/Results.ts]: you can use .join here, 

```js
let collisions = formArray.join(",");
```

File: cmd/server-manager/typescript/src/Results.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    private static showAllCollisions() {
+        let checkboxes = $(".event-checkbox");
+
+        checkboxes.bootstrapSwitch("state", true);
+
+        $("#collision-form").submit()
+    }
+
+    private static hideAllCollisions() {
+        console.log("hide all");
```
Comment (cj123) [inline cmd/server-manager/typescript/src/Results.ts]: console.log("hi")

File: results.go [hunk 3]
```diff
@@ -921,19 +925,7 @@ type resultsViewTemplateVars struct {
 }
 
 func (rh *ResultsHandler) view(w http.ResponseWriter, r *http.Request) {
-	var result *SessionResults
```
Comment (Hecrer) [inline results.go]: Move this back, copy to both places

File: results.go [hunk 4]
```diff
@@ -1016,6 +1008,121 @@ func (rh *ResultsHandler) edit(w http.ResponseWriter, r *http.Request) {
...
+
+			collisionsToLoad = append(collisionsToLoad, collisionNum)
+		}
+	}
+
+	trackMapData, err := LoadTrackMapData(result.TrackName, result.TrackConfig)
+
+	if err != nil {
+		logrus.WithError(err).Error("could not load track map data")
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
```
Comment (Hecrer) [inline results.go]: Need to return

File: results.go [hunk 5]
```diff
@@ -1016,6 +1008,121 @@ func (rh *ResultsHandler) edit(w http.ResponseWriter, r *http.Request) {
...
+func (c *circle) ColorModel() color.Model {
+	return color.AlphaModel
+}
+
+func (c *circle) Bounds() image.Rectangle {
+	return image.Rect(c.p.X-c.r, c.p.Y-c.r, c.p.X+c.r, c.p.Y+c.r)
+}
+
+func (c *circle) At(x, y int) color.Color {
+	xx, yy, rr := float64(x-c.p.X)+0.5, float64(y-c.p.Y)+0.5, float64(c.r)
```
Comment (cj123) [inline results.go]: this could just return c.color and avoid the mind bending mathematics

**Pull request merged**